NBXplorer.Client/ExplorerClient.cs

```diff
@@ -125,9 +125,14 @@ public Task<UTXOChanges> GetUTXOsAsync(DerivationStrategyBase extKey, Cancellati
 			return GetUTXOsAsync(TrackedSource.Create(extKey), cancellation);
 		}
```

Comment:
I would prefer to not do this changes. This is over abstraction IMHO.

Reply:
`GetTransactionAsync` is the only one needed now

Reply:
This is also completely useless as NBitcoin already serialize/deserialize the correct type.